### PR TITLE
Update help text for Google/Office 365 OAuth

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Update help text for Google/Office 365 OAuth.

Thanks @esethna for review in #3713 .

- [X] Includes text changes and localization files updated

### CODE REVIEW

**Comment by jasonblais** [comment]
Hmm, not sure why the build is failing?

**Comment by coreyhulen** [comment]
+1

**Comment by coreyhulen** [comment]
@jasonblais 

you need to escape the `'` single quotes in the jsx files or change to `"` double quotes.  If you change to double quotes then in it would need to be escaped in the .json files.

ERROR in ./components/admin_console/oauth_settings.jsx
Module build failed: SyntaxError: /var/lib/jenkins/jobs/mattermost-platform-pr/workspace@2/src/github.com/mattermost/platform/webapp/components/admin_console/oauth_settings.jsx: Unexpected token (388:310)
  386 |                 <FormattedHTMLMessage
  387 |                     id='admin.google.EnableHtmlDesc'
> 388 |                     defaultMessage="<ol><li><a href='https://accounts.google.com/login'>Log in</a> to your Google account.</li><li>Go to <a

**Comment by coreyhulen** [comment]
`defaultMessage="` should be `defaultMessage='` otherwise eslint will complain.

**Comment by coreyhulen** [comment]
you can see the error details at https://build.mattermost.com/job/mattermost-platform-pr/1956/console

File: webapp/i18n/en.json [hunk 1]
```diff
@@ -274,7 +274,7 @@
...
...evelopers.google.com'>https://console.developers.google.com</a>, click <strong>Credentials</strong> in the left hand sidebar and enter "Mattermost - your-company-name" as the <strong>Project Name</strong>, then click <strong>Create</strong>.</li><li>Click the <strong>OAuth consent screen</strong> header and enter "Mattermost" as the <strong>Product name shown to users</strong>, then click <strong>Save</strong>.</li><li>Under the <strong>Credentials</strong> header, click <strong>Create credentials</strong>, choose <strong>OAuth client ID</strong> and select <strong>Web Application</strong>.</li><li>Under <strong>Restrictions</strong> and <strong>Authorized redirect URIs</strong> enter <strong>your-mattermost-url/signup/google/complete</strong> (example: http://localhost:8065/signup/google/complete). Click <strong>Create</strong>.</li><li>Paste the <strong>Client ID<strong> and <strong>Client Secret<strong> to the fields below, then click <strong>Save</strong>.</li><li>Finally, go to <a href='https://console.developers.google.com/apis/api/plus/overview'>Google+ API</a> and click <strong>Enable</strong>. This might take a few minutes to propagate through Google's systems.</li></ol>",
```
Comment (enahum) [inline webapp/i18n/en.json]: this is failing cause the " character in the string should be escaped like this \"

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -284,7 +284,7 @@
...
+  "admin.office365.EnableHtmlDesc": "<ol><li><a href='https://login.microsoftonline.com/'>Log in</a> to your Microsoft or Office 365 account. Make sure it's the account on the same <a href='https://msdn.microsoft.com/en-us/library/azure/jj573650.aspx#Anchor_0'>tenant</a> that you would like users to log in with.</li><li>Go to <a href='https://apps.dev.microsoft.com'>https://apps.dev.microsoft.com</a>, click <strong>Go to app list</strong> > <strong>Add an app</strong> and use "Mattermost - your-company-name" as the <strong>Application Name</strong>.</li><li>Under <strong>Application Secrets</strong>, click <strong>Generate New Password</strong> and paste it to the <strong>Application Secret Password</strong> field below.</li><li>Under <strong>Platforms</strong>, click <strong>Add Platform</strong>, choose <strong>Web</strong> and enter <strong>your-mattermost-url/signup/office365/complete</strong> (example: http://localhost:8065/signup/office365/complete) under <strong>Redirect URIs</strong>. Also uncheck <strong>Allow Implicit Flow</strong>.</li><li>Finally, click <strong>Save</strong> and then paste the <strong>Application ID</strong> below.</li></ol>",
```
Comment (enahum) [inline webapp/i18n/en.json]: same here

**Comment by enahum** [comment]
+1

**Pull request merged**